lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -101,6 +102,11 @@ UsdMaya_ReadJob::~UsdMaya_ReadJob() { }
 
 bool UsdMaya_ReadJob::Read(std::vector<MDagPath>* addedDagPaths)
 {
+    // When we are called from PrimUpdaterManager we should already have
+    // a computation scope. If we are called from elsewhere don't show any
+    // progress bar here.
+    MayaUsd::ProgressBarScope progressBar(16);
```

Comment:
Using this constructor (just a steps number) will add to an existing (if one exists) progress bar. If no progress bar exists, this does nothing.